lib/src/migrators/module.dart

```diff
@@ -245,9 +245,11 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
     var hiddenByUrl = <Uri, Set<MemberDeclaration>>{};
     for (var declaration in references.globalDeclarations) {
       var private = declaration.name.startsWith('-');
+
       // Whether this member will be exposed by the regular entrypoint.
-      var visibleAtEntrypoint = declaration.sourceUrl == entrypoint ||
-          (_shouldForward(declaration.name) && !private);
+      var visibleAtEntrypoint = (declaration.sourceUrl == entrypoint ||
+              _shouldForward(declaration.name)) &&
+          !private;
```

Comment:
```suggestion
      var visibleAtEntrypoint = !private &&
          (declaration.sourceUrl == entrypoint || _shouldForward(declaration.name));
```

Nit: I think this would read a bit better (and possibly be a bit more efficient).